mypy/fscache.py

```diff
@@ -205,7 +205,7 @@ def isfile_case(self, path: str, prefix: str) -> bool:
         if sys.platform == "linux":
             # Assume that the file system on Linux is case sensitive
             return self.isfile(path)
-        if not self.isfile(path):
+        if not self.isfile(path):  # type: ignore[unreachable, unused-ignore]
```

Comment:
y are these errors ignored

Reply:
because they only appear on linux, and not on windows, easiest way to fix it for now, till I implement append baseline.